LEDBlinker: NUCLEO_F401RE+ST BLE NRG: Build errors - GCC OoB#3

Version: https://github.com/ARMmbed/mbed-os-example-ble/tree/mbed-os-5.1.0-rc3
OS: Mac 10.11.6
Toolchain: GCC_ARM
Target: NUCLEO_F401RE + ST BLUE NRG
Test: LEDBlinker

```
Compile: main.cpp
[Error] main.cpp@177,48: no matching function for call to 'events::EventQueue::post_every(void (&)(), int)'
[ERROR] ./source/main.cpp: In function 'int main()':
./source/main.cpp:177:48: error: no matching function for call to 'events::EventQueue::post_every(void (&)(), int)'
eventQueue.post_every(periodicCallback, 500);
^
In file included from ./mbed-events/events.h:19:0,
from ./source/main.cpp:17:
./mbed-events/EventQueue.h:176:9: note: candidate: template<class F> int events::EventQueue::post_every(int, F)
int post_every(int ms, F f) {
^
./mbed-events/EventQueue.h:176:9: note:   template argument deduction/substitution failed:
./source/main.cpp:177:48: note:   cannot convert 'periodicCallback' (type 'void()') to type 'int'
eventQueue.post_every(periodicCallback, 500);
^
In file included from ./mbed-events/events.h:19:0,
from ./source/main.cpp:17:
./mbed-events/EventQueue.h:190:9: note: candidate: template<class F, class A0> int events::EventQueue::post_every(int, F, A0)
int post_every(int ms, F f, A0 a0) {
^
./mbed-events/EventQueue.h:190:9: note:   template argument deduction/substitution failed:
./source/main.cpp:177:48: note:   cannot convert 'periodicCallback' (type 'void()') to type 'int'
eventQueue.post_every(periodicCallback, 500);
^
In file included from ./mbed-events/events.h:19:0,
from ./source/main.cpp:17:
./mbed-events/EventQueue.h:195:9: note: candidate: template<class F, class A0, class A1> int events::EventQueue::post_every(int, F, A0, A1)
int post_every(int ms, F f, A0 a0, A1 a1) {
^
./mbed-events/EventQueue.h:195:9: note:   template argument deduction/substitution failed:
./source/main.cpp:177:48: note:   cannot convert 'periodicCallback' (type 'void()') to type 'int'
eventQueue.post_every(periodicCallback, 500);
^
In file included from ./mbed-events/events.h:19:0,
from ./source/main.cpp:17:
./mbed-events/EventQueue.h:200:9: note: candidate: template<class F, class A0, class A1, class A2> int events::EventQueue::post_every(int, F, A0, A1, A2)
int post_every(int ms, F f, A0 a0, A1 a1, A2 a2) {
^
./mbed-events/EventQueue.h:200:9: note:   template argument deduction/substitution failed:
./source/main.cpp:177:48: note:   cannot convert 'periodicCallback' (type 'void()') to type 'int'
eventQueue.post_every(periodicCallback, 500);
^
In file included from ./mbed-events/events.h:19:0,
from ./source/main.cpp:17:
./mbed-events/EventQueue.h:205:9: note: candidate: template<class F, class A0, class A1, class A2, class A3> int events::EventQueue::post_every(int, F, A0, A1, A2, A3)
int post_every(int ms, F f, A0 a0, A1 a1, A2 a2, A3 a3) {
^
./mbed-events/EventQueue.h:205:9: note:   template argument deduction/substitution failed:
./source/main.cpp:177:48: note:   cannot convert 'periodicCallback' (type 'void()') to type 'int'
eventQueue.post_every(periodicCallback, 500);
^
In file included from ./mbed-events/events.h:19:0,
from ./source/main.cpp:17:
./mbed-events/EventQueue.h:210:9: note: candidate: template<class F, class A0, class A1, class A2, class A3, class A4> int events::EventQueue::post_every(int, F, A0, A1, A2, A3, A4)
int post_every(int ms, F f, A0 a0, A1 a1, A2 a2, A3 a3, A4 a4) {
^
./mbed-events/EventQueue.h:210:9: note:   template argument deduction/substitution failed:
./source/main.cpp:177:48: note:   cannot convert 'periodicCallback' (type 'void()') to type 'int'
eventQueue.post_every(periodicCallback, 500);
^

[mbed] ERROR: "python" returned error code 1.
[mbed] ERROR: Command "python -u /Users/barsza01/devel/mbed/mbed-os-example-ble/BLE_LEDBlinker/mbed-os/tools/make.py -t GCC_ARM -m NUCLEO_F401RE --source . --build ./.build/NUCLEO_F401RE/GCC_ARM" in "/Users/barsza01/devel/mbed/mbed-os-example-ble/BLE_LEDBlinker"

---
```

--- a/BLE_LEDBlinker/source/main.cpp
+++ b/BLE_LEDBlinker/source/main.cpp
@@ -174,7 +174,7 @@ void scheduleBleEventsProcessing(BLE::OnEventsToProcessCallbackContext* context)
 int main()
 {
     triggerLedCharacteristic = false;
-    eventQueue.post_every(periodicCallback, 500);
+    eventQueue.post_every(500, periodicCallback);
 
     BLE &ble = BLE::Instance();
     ble.onEventsToProcess(scheduleBleEventsProcessing);

